docs/process-for-changing-ion-schema.md

```diff
@@ -0,0 +1,177 @@
+# Process for changing the Ion Schema Specification
+
+This document describes the process to evolve the Ion Schema Language. It only applies to the Ion Schema Language itself; feature proposals for each of the Ion Schema libraries and related projects are out of scope.
+
+Without Ion Schema specification versions, almost any functional change to the Ion Schema specification is a potentially breaking change for Ion Schema users. When new features are added to the Ion Schema specification, prior releases of Ion Schema implementations will not have support for those features, causing a schema document to be interpreted differently depending on what version of an Ion Schema library is being used. To ensure that a schema document has consistent behavior across all releases of all implementations, new features are grouped into new versions of the Ion Schema specification. (For more details about Ion Schema versions, see [ISL Versioning](https://amazon-ion.github.io/ion-schema/docs/isl-versioning).)
+
+## Ion Schema Language Evolution Goals
+
+The following are the goals that inform the rest of this document.
+
+* The design work for Ion Schema should be open source.
+* The process should be lightweight and should not be an encumbrance to evolving Ion Schema.
+* Don’t let perfect be the enemy of good—the most important part of vetting a proposed change is ensuring that it is *not bad*.
```

Comment:
In the body of the document you explicitly assert that we evaluate whether something is good enough in order to advance through the process. Should we consider reframing those assertions to better align with this goal of the document?

Reply:
> Should we consider reframing those assertions to better align with this goal of the document?

Can you clarify what you mean by that?